apps/drain-safe/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "drain-safe",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@gnosis.pm/safe-apps-react-sdk": "2.2.0",
+    "@gnosis.pm/safe-react-components": "^0.5.0",
+    "@material-ui/core": "^4.11.0",
```

Comment:
I think there is no direct use of this library in this app, so we could leave the version managing to safe-react-components?

Reply:
it's a peer dependency: https://github.com/gnosis/safe-react-components/blob/development/package.json#L85

Reply:
so it has to be present

Reply:
Restored 👌